libs/datasets/timeseries.py

```diff
@@ -1374,9 +1374,8 @@ def write_to_wide_dates_csv(self, path_wide_dates: pathlib.Path, path_static: pa
         # that are now over 100M. Unfortunately this also writes unnecessarily high precision for
         # floats but I don't see an easy solution with to_csv float_format.
         # https://trello.com/c/aDGn57Df/1192-change-combined-data-from-csv-to-parquet will remove
-        # the need to format values as strings.
-        csv_buf = wide_df.to_csv(index=True, float_format="%.9g")
```

Comment:
Pandas can handle gzip compression under the hood without us having to use the `gzip` library, or any sort of file handler, so I've opted for that for simplicity. 

I'm not sure what the specific reason was for writing to a csv object and then using pathlib to write that test to a file, but writing the csv directly using pandas (and gzip compression) seems to work fine.